purchases-js-hybrid-mappings/src/purchases-common.ts

```diff
@@ -37,9 +40,14 @@ export class PurchasesCommon {
       flavor: configuration.flavor,
       version: configuration.flavorVersion,
     });
+    let httpConfig: HttpConfig | undefined = undefined;
+    if (PurchasesCommon.proxyUrl) {
+      httpConfig = { proxyURL: PurchasesCommon.proxyUrl };
+    }
     const purchasesInstance = Purchases.configure(
       configuration.apiKey,
       configuration.appUserId || Purchases.generateRevenueCatAnonymousAppUserId(),
+      httpConfig,
```

Comment:
I'm not too sure about this, but it looks to me like the `httpConfig` parameter cannot be `undefined`, as declared in the `configure` method of `Purchases` that I checked [here](https://github.com/RevenueCat/purchases-js/blob/25af11e92612ca6d23e709c0b843c5a02aa9a250/src/main.ts#L205-L210).
```
static configure(
    apiKey: string,
    appUserId: string,
    httpConfig: HttpConfig = defaultHttpConfig,
    flags: FlagsConfig = defaultFlagsConfig,
  ): Purchases
```

Reply:
I believe `undefined` is the "default" value if the parameter is not passed, in which case, it will pick the default value provided, in this case `defaultHttpConfig`.

Reply:
mmm I see your point. I guess this is different from how default method parameters work in Swift 😅 
Thanks for the clarification!